app/services/process_feed_service.rb

```diff
@@ -163,7 +163,7 @@ def mentions_from_xml(parent, xml)
 
         url = Addressable::URI.parse(link['href'])
 
-        mentioned_account = if TagManager.instance.local_domain?(url.host)
+        mentioned_account = if TagManager.instance.web_domain?(url.host)
```

Comment:
Actually no, the URI in the “mentioned” links from other people's toots seem to be full URLs to the web front-end, thus using WEB_DOMAIN and not LOCAL_DOMAIN